src/components/SingleList.jsx

```diff
@@ -1,19 +1,42 @@
 import { useNavigate } from 'react-router-dom';
 
 import './SingleList.css';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faTrashCan } from '@fortawesome/free-regular-svg-icons';
+import { deleteSingleList } from '../api/firebase';
+import { useAuth } from '../api';
 
 export function SingleList({ name, path, setListPath }) {
+	const { user } = useAuth();
+
 	const navigate = useNavigate();
 
 	function handleClick() {
 		setListPath(path);
 		navigate('list');
 	}
-
+	async function deleteSelectedList() {
+		if (
+			window.confirm(
+				`Are you sure you want to delete the ${name} list? This action cannot be undone and all items in this list will be deleted.`,
+			)
+		) {
+			try {
+				const result = await deleteSingleList(user.email, path);
+				alert(result.message);
+				setListPath(null);
+			} catch (error) {
+				alert(`Failed to delete this list: ${error.message}`);
+			}
+		}
+	}
 	return (
 		<div className="SingleList">
 			<li>
 				<button onClick={handleClick}>{name}</button>
+				<button aria-label={`Delete ${name}`} onClick={deleteSelectedList}>
```

Comment:
If you put `className="delete-btn"` here, when this issue and my issue for the `List` styling is merged, the same styling I applied to the delete buttons on the `List` page will be applied.

I made a `delete-btn` global rule in the `index.css`, as I knew you were adding a delete button on the Home page, and that we would want the styling to be the same.

Reply:
Hi @rachelspencer, I have removed the 'Delete' text and will also add the className="delete-btn" to maintain consistency in our styling!